shared/src/test/scala/squants/thermal/TemperatureSpec.scala

```diff
@@ -55,6 +55,18 @@ class TemperatureSpec extends FlatSpec with Matchers {
     Temperature("ZZ F").failed.get should be(QuantityParseException("Unable to parse Temperature", "ZZ F"))
   }
 
+  they should "be flexible in parsing strings with regard to degree symbol and whitespace" in {
+    Temperature("10.22 f").get should be (Fahrenheit(10.22))
+    Temperature("10.22 °f").get should be (Fahrenheit(10.22))
+    Temperature("10.22  °f").get should be (Fahrenheit(10.22))
```

Comment:
We already allow this syntax with the current Temperature parsing regex. I was just adding tests to make sure I didn't break anything while I was changing the regex.